From: Maximilian Weigand <mweigand@mweigand.net>
Date: Fri, 5 Jul 2024 19:36:53 +0800
Subject: dts: pinenote: always leave vdd_gpu_npu: DCDC_REG2 on

We had problems with the gpu being permanently disabled if not used
within 30 seconds after boot....
---
 arch/arm64/boot/dts/rockchip/rk3566-pinenote.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3566-pinenote.dtsi b/arch/arm64/boot/dts/rockchip/rk3566-pinenote.dtsi
index a501d43..f04a5ca 100644
--- a/arch/arm64/boot/dts/rockchip/rk3566-pinenote.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3566-pinenote.dtsi
@@ -425,6 +425,7 @@ regulator-state-mem {
 
 			vdd_gpu_npu: DCDC_REG2 {
 				regulator-name = "vdd_gpu_npu";
+				regulator-always-on;
 				regulator-min-microvolt = <500000>;
 				regulator-max-microvolt = <1350000>;
 				regulator-ramp-delay = <6001>;
